Formula/b/beast.rb

```diff
@@ -49,7 +49,14 @@ def install
     inreplace "testUCRelaxedClockLogNormal.xml", 'chainLength="10000000"',
                                                  'chainLength="100000"'
 
-    system "#{bin}/beast", "testUCRelaxedClockLogNormal.xml"
+    # OpenCL does not work on ephemeral ARM CI.
+    ephemeral_arm = ENV["HOMEBREW_GITHUB_ACTIONS"].present? &&
+                    Hardware::CPU.arm? &&
+                    OS.mac? &&
+                    MacOS.version > :big_sur
+    args = ephemeral_arm ? ["-beagle_CPU"] : []
```

Comment:
This is more of a VM thing - it should be reproducible locally in one

```suggestion
    # OpenCL is not supported on virtualized arm64 macOS
    virtual_arm_mac = OS.mac? &&
                    Hardware::CPU.arm? &&
                    Hardware::CPU.virtualized?
    args = virtual_arm_mac ? ["-beagle_CPU"] : []
```

Reply:
I'm probably going to have to skip running command as `beagle` is still trying to check GPU and I don't know if there is a way to avoid. So may end up with:
```suggestion
    # OpenCL is not supported on virtualized arm64 macOS
    return if OS.mac? && Hardware::CPU.arm? && Hardware::CPU.virtualized?
````